README.md

```diff
@@ -33,6 +33,7 @@
 - [Real Time Multiplayer in HTML5](http://buildnewgames.com/real-time-multiplayer/) - Sven Bergström's multiplayer game development on the web guide.
 - [Replication in Networked Games](https://0fps.net/2014/02/10/replication-in-networked-games-overview-part-1/) - Mikola Lysenko's replication articles for JavaScript-based multiplayer.
 - [Rollback Networking in INVERSUS](http://blog.hypersect.com/rollback-networking-in-inversus/) - Ryan Juckett's post on a peer-to-peer rollback system in multiplayer.
+- [Delta Rollback: New optimizations for Rollback Netcode](https://medium.com/@david.dehaene/delta-rollback-new-optimizations-for-rollback-netcode-7d283d56e54b) - Rollback Netcode optimizations for a 10 players action game.
```

Comment:
Thank you! I'll take a look and hopefully merge when I have a bit free time soon, really appreciate your contribution!

Reply:
hey @BimDav, now I get to go through the article and also checked out GitLab repo then I came across this YouTube playlist: https://www.youtube.com/playlist?list=PLCBLMvLIundBXwTa6gwlOUNc29_9btoir
I didn't go through the whole playlist, so I figured I might ask you directly since you're the author of all of these good stuff :)
should we rather have a link to the YouTube playlist or just a link to this main article in the articles section?
what would you suggest and why?

Reply:
hey, I'm not the author of these videos ;) but they really cover well the basics of Rollback Netcode, so it definitely makes sense to add them as well! The article in the PR is more about new approach / optimizations to Rollback Netcode.

Reply:
what was I thinking?! yeah, I should never try to do so much in the middle of the night :D
sounds good, I'll get this in and go watch the playlist later to potentially add to the list.
thank you for your contribution to the community! :)